archstrike-testing/python2-rekall/PKGBUILD

```diff
@@ -0,0 +1,47 @@
+# Maintainer: ArchStrike <team@archstrike.org>
+# Contributor: strayArch <j.a.stronz@gmail.com>
+
+buildarch=1
+
+_pkgname="rekall"
+pkgname="python2-${_pkgname}"
+pkgver="1.6.0"
+pkgrel=1
+pkgdesc="A Memory Forensic Framework"
+url="https://www.rekall-forensic.com/"
+arch=("any")
+groups=("archstrike" "archstrike-analysis" "archstrike-forensics")
+license=("GPL2")
+depends=("python2")
+makedepends=("python2-setuptools" "python2-virtualenv")
+replaces=("${pkgname}<${pkgver}")
```

Comment:
don't quote me on it, but i think old versions of a package are removed automatically when a new version is installed. this might not be necessary